screens/Home.tsx

```diff
@@ -221,23 +252,19 @@ export default function Home({ navigation }: HomeProps) {
         >
           <JournalEntry entry={item} />
         </TouchableOpacity>
-        {/* <Button
-          title="Delete"
-          color="red"
-          onPress={() => {
-            dispatch(removeEntry(item.id));
-          }}
-        /> */}
+        
       </View>
     );
   };
 
   const { width, height } = Dimensions.get("window");
 
   return (
-    <View style={{ flex: 1, backgroundColor: "#ededed" }}>
-      <View style={{flex: 8}}>
+    <View style={{ flex: 1, paddingTop:80 }}>
+      <View style={{flex: 2, paddingBottom:20}}>
       <Carousel
+      layout="tinder"
```

Comment:
tinder layout aakumbo today's card is at the bottom.
I think we should go back to the previous layout.

Reply:
Ok